tests/framework/src/runner.rs

```diff
@@ -235,12 +235,12 @@ impl TestRunner {
                         _ => panic!("MouseWheel: expected only one of 'lines' or 'pixels'"),
                     },
                 },
-                AutomatedEvent::KeyDown { key_code } => PlayerEvent::KeyDown {
-                    key_code: KeyCode::from_code(*key_code),
+                AutomatedEvent::KeyDown { key } => PlayerEvent::KeyDown {
+                    key_code: automated_key_to_key_code(*key),
                     key_char: None,
```

Comment:
I guess we could actually do something useful here in the future :tada:

Reply:
You bet! I'm cookin' somethin'